dev/com.ibm.ws.jbatch.open_fat/publish/servers/batchFAT/bootstrap.properties

```diff
@@ -7,5 +7,11 @@ com.ibm.ws.logging.trace.specification=*=info=enabled\
 :com.ibm.ws.jbatch.open_fat=all=enabled\
 :batch.fat.*=all=enabled
 
+websphere.java.security.exempt=true
```

Comment:
We cannot disable security for all other buckets that use this `batchFAT` server configuration.